### PR TITLE
Generate various class synopses from stubs - part 1

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/simplexml/simplexmlelement.xml [hunk 1]
```diff
@@ -20,34 +20,68 @@
...
+       <entry>&Version;</entry>
+       <entry>&Description;</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>8.0.0</entry>
+       <entry>
+        <classname>SimpleXMLElement</classname> implements
+        <interfacename>Stringable</interfacename> now.
```
Comment (cmb69) [inline reference/simplexml/simplexmlelement.xml:72]: It seems to me that `Countable` and `RecursiveIterator` are also only implemented as of PHP 8.0.0. Apparently, these interfaces have been implemented by `SimpleXMLIterator` prior to that version.

**Review by cmb69 — COMMENTED** [review]
Thank you! Except for the SimpleXMLElement/SimpleXMLIterator interface implementations, this looks good to me.

**Comment by cmb69** [comment]
I think that SimpleXMLIterator needs a changelog entry as well.

**Comment by kocsismate** [comment]
> I think that SimpleXMLIterator needs a changelog entry as well.

SimpleXMLIterator still (indirectly) implements these interfaces, so IMO there's not much change from a child class POV. Or what do you think the change log entry should mention? Although, I'vejustrealized that the newly implemented Stringable interface could be mentioned.

**Review by cmb69 — APPROVED** [review]
Ah, right SimpleXMLIterator extends SimpleXMLElement, so everything is fine, in my opinion.

**Pull request merged**